lua/nvim-tree/renderer/components/git.lua

```diff
@@ -60,19 +60,19 @@ local function warn_status(git_status)
 end
 
 local function get_icons_(node)
-  local git_statuses = explorer_common.get_git_status(node)
-  if git_statuses == nil then
+  local git_status = explorer_common.get_git_status(node)
```

Comment:
I was actually thinking we could use `git_statuses` everwhere as it is a much better variable name, similar to `iconss`.

However... looking through the whole codebase we are using `git_status` _everywhere_ so I guess that change is outside of the scope of this PR.

Reply:
Ahhh sorry, I misunderstood. Perhaps I'll open a PR right after this one specifically for git_statuses?

Reply:
If you're keen I would be most grateful! There are a lot of usages...